### PR TITLE
Topics: add new topics (dongcarl)

### PR DESCRIPTION
**Pull request by harding**
Eight proposed pages for the new `/en/topics/` section of the site.  Suggested reviewer is @dongcarl 

Note: tests expected to fail on the final "production" test because of the FIXME link pointing to a topic that's part of a different PR.  That's the last test we run, so if it's the only failure, this should be ok.

### CODE REVIEW

File: _topics/en/hwi.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Hardware Wallet Interaction (HWI)
```
Comment (achow101) [inline _topics/en/hwi.md]: s/Interaction/Interface. we renamed it some time ago.

File: _topics/en/hwi.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Wallet Collaboration Tools
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Hardware Wallet Interaction (HWI)** is a set of Python scripts used
```
Comment (achow101) [inline _topics/en/hwi.md]: Would be more accurate to say

> is a Python library and command line tool

File: _topics/en/psbt.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Partially-Signed Bitcoin Transactions (PSBTs)** are a data format
+  that allows wallets and other tools to exchange information about a
+  Bitcoin transaction and the signatures necessary to complete it.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  A PSBT can be created with just information about what outputs are to
```
Comment (achow101) [inline _topics/en/psbt.md]: This sentence isn't entirely true. PSBT shouldn't really be used with 0-input txs to be passed around for funding. We specifically avoid doing this in Core.

File: _topics/en/minisketch.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Minisketch** is a library implementing an algorithm that allows
+  efficient set reconcilliation of announcements in gossip protocols,
+  such as those used in Bitcoin and LN.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  The type of set reconcilliation provided by minisketch underlies the
+  proposed [Erlay][topic erlay] enhancement to the Bitcoin gossip
```
Comment (dongcarl) [inline _topics/en/minisketch.md]: Perhaps we should mention that the proposal is mainly for transaction relay?

File: _topics/en/reproducible-builds.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  This means no one person or computer needs to be trusted to produce
+  the executable binaries that most people use.  Additionally, people
+  who compile the software themselves can ensure that they received an
+  executable that's identical to what other people received, helping to
+  ensure their software will remain compatible with other software, e.g.
+  a full node staying in consensus with the rest of the network.
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
```
[the remark this answers is not in the document]
Comment (harding) [reply _topics/en/reproducible-builds.md]: The PR is a good addition, but I'd prefer not to make presentations or videos into primary sources, even when someone I like is rockin' the mic.  I think we want the primary sources to be documents, diffs, and other things that can be easily searched and skimmed.  Would it be ok if I put the video in the `see_also` section instead?

[the remark this answers is not in the document]
Comment (dongcarl) [reply _topics/en/reproducible-builds.md]: Of course! :-)

File: _topics/en/miniscript.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  including determining what data is necessary to create a witness that
+  fulfills the script and allows any bitcoins protected by the script to
+  be spent.  With miniscript telling the wallet what it needs to do,
+  wallet developers don't need to write new code when they switch from
+  one script template to another.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  This automation for the large range of scripts supported by miniscript
+  allows wallets to be much more dynamic about the scripts they use,
```
Comment (dongcarl) [inline _topics/en/miniscript.md]: Perhaps something like "Miniscript's structured representation of a large range of Bitcoin scripts allows..."

**Comment by harding** [comment]
@dongcarl thank you so much for the reviews!

**Comment by jnewbery** [comment]
Ready for merge when _erlay_ (#246) is merged

**Comment by jnewbery** [comment]
Merged in #246

**Pull request closed without merging**